BasiliskII/src/rom_patches.cpp

```diff
@@ -1008,6 +1008,179 @@ static bool patch_rom_classic(void)
+	wp = (uint16 *)(ROMBaseHost + 0x10a8);
+	*wp = htons(MacScreenHeight);
+	wp = (uint16 *)(ROMBaseHost + 0x10ac);
+	*wp = htons(MacScreenWidth);
+	wp = (uint16 *)(ROMBaseHost + 0x10b0);
+	*wp = htons(MacScreenWidth / 8);
+	lp = (uint32 *)(ROMBaseHost + 0x10b4);
+	*lp = htonl(MacScreenWidth * MacScreenHeight / 8);// # of bytes in screen
+
+	// sad mac, mac icon
+	lp = (uint32 *)(ROMBaseHost + 0x118a);
+	*lp = htonl(MacFrameBaseMac24Bit
+							+ (((MacScreenHeight / 4) * 2 - 25) * MacScreenWidth
+								+ (MacScreenWidth / 2 - 16))
+								/ 8);
+	// sad mac, frown
+	lp = (uint32 *)(ROMBaseHost + 0x1198);
+	*lp = htonl(MacFrameBaseMac24Bit
+							+ (((MacScreenHeight / 4) * 2 - 19) * MacScreenWidth
+								+ (MacScreenWidth / 2 - 8))
+								/ 8);
+	wp = (uint16 *)(ROMBaseHost + 0x11b0);
+	*wp = htons(MacScreenWidth / 8);
+
+	// blink floppy and sadmac, position
+	wp = (uint16 *)(ROMBaseHost + 0x11d8);
+	*wp = htons(MacScreenWidth / 8);
+	wp = (uint16 *)(ROMBaseHost + 0x11ea);
+	*wp = htons(MacScreenWidth / 8);
+
+	// cursor handling
+	if (MacScreenWidth >= 1024)
+	{
+		wp = (uint16 *)(ROMBaseHost + 0x18dfe); // start to patch P_HideCursor routine
+		*wp++ = htons(0x4eb9); /* JSR */
+		lp = (uint32 *)wp;
+		*lp++ = htonl(patchCodeBaseMac);
+		wp = (uint16 *)lp;
+
+		*patchwp++ = htons(0x41f8); /* Lea.L (CrsrSave),A0 */
+		patchCodeBaseMac += 2;
+		*patchwp++ = htons(0x088c);
+		patchCodeBaseMac += 2;
+
+		*patchwp++ = htons(0x203c); /* MOVE.L #$x,D0 */
+		patchCodeBaseMac += 2;
+		patchlp = (uint32 *)patchwp;
+		*patchlp++ = htonl(MacScreenWidth / 8);
+		patchCodeBaseMac += 4;
+		patchwp = (uint16 *)patchlp;
+
+		*patchwp++ = htons(0x4e75); /* RTS */
+		patchCodeBaseMac += 2;
+		patchlp = (uint32 *)patchwp; // keep in sync
+	} else
+	{
+		// P_HideCursor
+		wp = (uint16 *)(ROMBaseHost + 0x18e02);
+		*wp = htons(0x7000 + (MacScreenWidth / 8));
+	} // end if (MacScreenWidth >= 1024)
+
+	wp = (uint16 *)(ROMBaseHost + 0x18e7a);
+	*wp = htons(MacScreenWidth - 32);
+	wp = (uint16 *)(ROMBaseHost + 0x18e80);
+	*wp = htons(MacScreenWidth - 32);
+	wp = (uint16 *)(ROMBaseHost + 0x18ea0);
+	*wp = htons(MacScreenHeight - 16);
+	wp = (uint16 *)(ROMBaseHost + 0x18ea6);
+	*wp = htons(MacScreenHeight);
+
+	if (MacScreenWidth >= 1024)
+	{
+		wp = (uint16 *)(ROMBaseHost + 0x18ec4); // start to patch P_ShowCursor routine
+		*wp++ = htons(0x4eb9); /* JSR */
+		lp = (uint32 *)wp;
+		*lp++ = htonl(patchCodeBaseMac);
+		wp = (uint16 *)lp;
+
+		*patchwp++ = htons(0x2a3c); /* MOVE.L #$x, D5 */
+		patchCodeBaseMac += 2;
+		patchlp = (uint32 *)patchwp;
+		*patchlp++ = htonl(MacScreenWidth / 8);
+		patchCodeBaseMac += 4;
+		patchwp = (uint16 *)patchlp;
+
+		*patchwp++ = htons(0xc2c5); /* MulU      D5, D1 */
+		patchCodeBaseMac += 2;
+
+		*patchwp++ = htons(0xd3c1); /* AddA.L    D1, A1 */
+		patchCodeBaseMac += 2;
+
+		*patchwp++ = htons(0x4e75); /* RTS */
+		patchCodeBaseMac += 2;
+		patchlp = (uint32 *)patchwp; // keep in sync
+	} else
```

Comment:
Nit: Weird formatting?

Reply:
![rom_patches cpp](https://user-images.githubusercontent.com/3475933/87614688-c2b67c00-c6de-11ea-8041-84f435972010.png)

They are tabs. It is consistent with the rest of the file.

Reply:
I got you. It is if/else. Done